repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/content-cards.ftl

```diff
@@ -1,5 +1,5 @@
 <#assign hst=JspTaglibs["http://www.hippoecm.org/jsp/hst/core"] >
-<#include "./link.ftl">
+<#import "./link.ftl" as hee>
```

Comment:
@thojava - I guess hash/prefix `hee` fixes FTL namespace conflict issue, but we may not be able to use `hee` for another import which again requires hash/prefix. How about naming `render` or `include` or anything appropriate for single macro based FTL templates like the following ?

```
File: repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/link.ftl
3: <#macro render link cssClassName>
```

```
File: repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/content-cards.ftl
2: <#import "./link.ftl" as link>
...
16:                             <@link.render link=contentCard.header cssClassName="nhsuk-card__link"/>
```
Thoughts ? Thanks!

Reply:
@smanivasag We can not use the `link` as namespace because it also conflicts with default library `link` tag.  I think we can just use the namespace as `hlink` ( stand for heelink ). 
I also introduce the way to group all macro into single `components.ftl` file then `<#import "../macros/components.ftl" as hee>` when need. You can review that under file guidance-main.ftl and listingpage-main.ftl

Reply:
@thojava - Sounds good, thanks 👍